tests/tst_ncio.F90

```diff
@@ -242,6 +242,12 @@ program tst_ncio
      stop 99
   end if
 
+  call read_vardata(dset, 'time', values_1d)
+  if (size(values_1d) .ne. 1 .or. values_1d(1) .ne. 0.) then
+     print *,'***time variable data read as 1d not correct...'
+     stop 99
+  end if
```

Comment:
I the netCDF and PIO builds, what I normally do is first (in the test) construct the test file with a bunch of values computed by some simple algorithm, like:
for (i=0; i<NUM_VALUES; i++)
   data[i] = i;

Then I read and check every single value.

Reply:
done